### ISSUE TITLE
[WIP][Sourcing] ACC-264.1(Level A) Fdp-table search icons label, UI type and properties are read incorrect

### ISSUE THREAD

**Issue by devika-devanand** [SAP/fundamental-ngx#11471]
#### Is this a bug, enhancement, or feature request?
Accessibility bug

#### Describe your proposal.
On accessing the ‘Search’ and ‘Refresh’ buttons of ‘Search’ input field using the JAWS reader, their labels and UI types are not conveyed to the users. Here the same properties of ‘Search’ field is being conveyed for the two buttons also which is incorrect.

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
fundamental-ngx version : 0.43.11

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
-Create an fdp-table which has the search and synchronize functionality in the toolbar.
-Activate the JAWS reader(JAWS Version 2024.2312.53 ILM) and navigate through the table search input box, search icon and synchronize icon.
![image](https://github.com/SAP/fundamental-ngx/assets/71935676/cbbe6112-d304-42e4-8c70-b6075e5235ed)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Labels, state, properties of all the UI elements should be conveyed correctly only once to the users when accessed.

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us here?

**Comment by droshev** [comment]
@devika-devanand any update?

**Comment by droshev** [comment]
@devika-devanand the ticket will be closed if there is no further updates

**Comment by droshev** [comment]
no activities => closing the issue
